### PR TITLE
Add pose model

### PR DESCRIPTION
**Pull request by Louis-Dupont**

### CODE REVIEW

**Comment by BloodAxe** [comment]
I suggest changing install command from
`!pip install super_gradients==3.3.1 scikit-learn gwpy &> /dev/null`

to 
`!pip install -qq super_gradients==3.3.1 scikit-learn`

**Comment by BloodAxe** [comment]
Results after first epochs in predict() shows no predictions. Maybe for sake of testing let it train a few epochs to ensure that after a few epochs when AP gets to reasonable score the predictions are actually rendered?

**Comment by BloodAxe** [comment]
This https://github.com/Deci-AI/super-gradients/pull/1607 should be merged before current PR

**Comment by Louis-Dupont** [comment]
I updated the code, but I am rerunning the notebook to make sure it's all right

File: notebooks/DEKR_PoseEstimationFineTuning.ipynb [hunk 1]
```diff
@@ -0,0 +1,1416 @@
...
+      ],
+      "metadata": {
+        "collapsed": false,
+        "id": "obAU90uggSAd"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!pip install -q super_gradients==3.4.1 scikit-learn"
```
Comment (BloodAxe) [inline notebooks/DEKR_PoseEstimationFineTuning.ipynb:25]: Pls update to 3.5 🙏

Comment (Louis-Dupont) [reply notebooks/DEKR_PoseEstimationFineTuning.ipynb]: [50bf234](https://github.com/Deci-AI/super-gradients/pull/1593/commits/50bf23491b00609f522d253cbf37efbb71eeb1b7)

**Review by BloodAxe — APPROVED** [review]
LGTM
